lib/health-indicators/typeorm/typeorm.health.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, Optional } from '@nestjs/common';
+import { Connection } from 'typeorm';
+import { promiseTimeout } from '../../utils';
+import { DatabaseHealthIndicator } from '../abstract/database-health-indicator';
+
+/**
+ * The TypeOrmeHealthIndicator contains health indicators
+ * which are used for health checks related to typeorm
+ */
+@Injectable()
+export class TypeOrmHealthIndicator extends DatabaseHealthIndicator {
```

Comment:
This would introduce a breaking change. I agree `TypeOrmHealthIndicator` is a better name, but I don't think it is a good idea to break the API with just one minor naming inconsistency. I think we should keep the name `DatabaseHealthIndicator`, but create an issue to rename it to `TypeOrmHealthIndicator` for the next major release.

Reply:
Agreed!